projectional/src/main/java/jetbrains/jetpad/projectional/cell/BaseProjectionalSynchronizer.java

```diff
@@ -66,6 +70,8 @@
   private DeleteHandler myDeleteHandler = DeleteHandler.EMPTY;
   private ContentKind<SourceItemT> myItemKind;
   private Function<SourceItemT, SourceItemT> myCloner;
+  private ListMap<ContentKind, ContentToItemMapper> myCompatibleContentKinds = new ListMap<>();
```

Comment:
In supportContentKind() you don't use "compatible" (supportCompatibleContentKind). It's assumed that it's compatible. So if we name the field myContentKinds we assume that they are myCompatibleContentKinds. Can we have not compatible content kinds?